### PR TITLE
Use validForDays from database or Config contract where appropriate

### PR DESCRIPTION
**Pull request by lukemelia**

### CODE REVIEW

File: packages/cardpay-sdk/sdk/scheduled-payment-module.ts [hunk 1]
```diff
@@ -1372,4 +1373,10 @@ export default class ScheduledPaymentModule {
       gasRangeInUSD,
     };
   }
+
+  async getValidForDays(): Promise<number> {
+    let configModuleAddress = await getAddress('usdStableCoinToken', this.ethersProvider);
```
Comment (FadhlanR) [inline packages/cardpay-sdk/sdk/scheduled-payment-module.ts:1378]: It is supposed to be `let configModuleAddress = await getAddress('scheduledPaymentConfig', this.ethersProvider);`.

File: packages/cardpay-sdk/sdk/scheduled-payment-module.ts [hunk 2]
```diff
@@ -1372,4 +1373,10 @@ export default class ScheduledPaymentModule {
...
+    const scheduledPaymentConfig = new Contract(configModuleAddress, ScheduledPaymentConfigABI, this.ethersProvider);
+    return await scheduledPaymentConfig.feeReceiver();
```
Comment (FadhlanR) [inline packages/cardpay-sdk/sdk/scheduled-payment-module.ts:1380]: It is supposed to be `scheduledPaymentConfig.validForDays()`.

**Review by FadhlanR — APPROVED** [review]
Looks great!

**Comment by lukemelia** [comment]
I'll wait for @jurgenwerk to be back to review before merging since he wrote most of the code I updated.

**Review by jurgenwerk — APPROVED** [review]
Looks good. I added a code comment to make it clearer why we introduced new params to the fetcher, please check if it makes sense.

**Comment by lukemelia** [comment]
>I added a code comment to make it clearer why we introduced new params to the fetcher, please check if it makes sense.
Looks good, thanks!

**Pull request merged**